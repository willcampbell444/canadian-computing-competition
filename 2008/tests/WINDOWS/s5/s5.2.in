5
9 9 5 8
10 8 7 9
9 10 8 9
10 10 10 10
8 8 9 10
