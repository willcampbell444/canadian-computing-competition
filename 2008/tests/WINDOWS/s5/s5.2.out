Patrick
Roland
Patrick
Roland
Patrick
